metrics/metrics.go

```diff
@@ -8,11 +8,13 @@ import (
 
 // Labels defines the labels that can be attached to the metrics.
 type Labels struct {
-	Source        DemandSource
-	RType         RequestType
-	PubID         string // exchange specific ID, so we cannot compile in values
-	CookieFlag    CookieFlag
-	RequestStatus RequestStatus
+	Source           DemandSource
+	RType            RequestType
+	PubID            string // exchange specific ID, so we cannot compile in values
+	CookieFlag       CookieFlag
+	RequestStatus    RequestStatus
+	DebugFlag        bool
+	AccountDebugFlag bool
```

Comment:
I think you'll be fine with a single debug flag, perhaps naming `DebugEnabed` or `DebugRequested` or perhaps even `Debug`. We don't care _how_ debug is enabled, just that it is _is_ enabled. With the implementation of the debug permissions, I can easily see how you arrived at this though. :)

Reply:
Ah this makes sense! I'll update accordingly.